app/containers/Button/index.js

```diff
@@ -0,0 +1,77 @@
+/**
+ *
+ * Button
```

Comment:
Is Button component still used?

Reply:
it is not used now, but it will be needed later